__tests__/lib/computeRollbackQuery.unit.ts

```diff
@@ -33,6 +33,17 @@ describe('computeRollbackQuery', () => {
           $set: { 'nested.key': 'value1' },
         });
       });
+
+      it('should set back the nested original array value', async () => {
```

Comment:
@pp0rtal Beware that you artificially increased the number of tests by writing twice this test (in commit 020236114af40927e4f04071c487f1ba218f6422 and in commit 020236114af40927e4f04071c487f1ba218f6422)